.env.example

```diff
@@ -2,6 +2,7 @@ SERVER_PORT=5000
 ENV=development
 
 LOG_OUTPUT=./server.log
+LOG_LEVEL=debug
 
 # see lib/logger.go
 LOG_LEVEL=info
```

Comment:
@dsmackie 
`LOG_LEVEL` is already shown in the example file in line 8. 
Your addition does the same thing. 
Can you remove either line 5 or line 8.